components/camel-aws-sns/src/main/java/org/apache/camel/component/aws/sns/SnsProducer.java

```diff
@@ -17,10 +17,7 @@
 package org.apache.camel.component.aws.sns;
 
 import java.nio.ByteBuffer;
-import java.util.Date;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Please avoid * imports.

Reply:
My intellij is on default setting and trying to be smart here :D. Do I need to change ?

Reply:
If you run the maven install with sourcecheck profile activated that imports will be reported as invalid. So it would be better to add explicit imports

Reply:
understood, done